<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>474</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>484</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>638</width>
    <height>474</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 		rgba(91, 91, 91, 255), stop:1 rgba(118, 118, 118, 255));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 85 20pt &quot;Bahnschrift&quot;;
color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string> Proyecto final</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#boton1{
color: #F0FF42;
font: 12pt &quot;Bahnschrift Condensed&quot;;
background-color: #3f3f3f;
border-radius: 10px;
border: 2px solid #54B435;
}
QPushButton:hover#boton1{
background-color: #55ffff;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Maximizar</string>
  </property>
   </widget>
   <widget class="QPushButton" name="boton2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#boton2{
color: ;
font: 12pt &quot;Bahnschrift Condensed&quot;;
background-color: ;
border-radius: 10px;
border: ;
}
QPushButton:hover#boton2{
background-color: ;
color: ;
}</string>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
